Resources/public/js/views/ez-serversideview.js

```diff
@@ -41,15 +41,91 @@ YUI.add('ez-serversideview', function (Y) {
              *
              * @event submitForm
              * @param {Node} form the Node object of the submitted form
+             * @param {Object} formData the serialzed form data including the
+             * used button to validate the form
              * @param {Event} originalEvent the original DOM submit event
              */
             this.fire('submitForm', {
                 form: e.target,
+                formData: this._serializeForm(e.target),
                 originalEvent: e,
             });
         },
 
         /**
+         * Serializes the form data so that it can be send with an AJAX request.
+         * It also checks which button was used (if any) to include it in the
+         * data.
+         *
+         * @method _serializeForm
+         * @param {Node} form
+         * @return {Object} key/value data of the form
+         */
+        _serializeForm: function (form) {
+            var focusedNode = form.get('ownerDocument').get('activeElement'),
+                data = {};
+
+            if ( this._isSubmitButton(focusedNode, form) ) {
+                data[focusedNode.getAttribute('name')] = "";
+            }
+
+            form.get('elements').each(function (field) {
+                var name = field.getAttribute('name'),
+                    type = field.get('type');
+
+                if ( !name ) {
+                    return;
+                }
+
+                /* jshint -W015 */
+                switch (type) {
+                    case 'button':
+                    case 'reset':
+                    case 'submit':
+                        break;
+                    case 'radio':
+                    case 'checkbox':
+                        if ( field.get('checked') ) {
+                            data[name] = field.get('value');
+                        }
+                        break;
+                    default:
```

Comment:
Maybe add `select` as well? Or at least add a comment to make it clear that it works the same as `input`.